sdk/turing/router/router.py

```diff
@@ -171,6 +171,16 @@ def list_versions(self) -> List['RouterVersion']:
         return [RouterVersion(environment_name=self.environment_name, name=self.name, **ver.to_dict())
                 for ver in response]
 
+    def create_version(self, config: RouterConfig) -> RouterVersion:
```

Comment:
Can this simply be a `create` method on the RouterVersion class? That's consistent with the API. Although the UI is designed to be Save vs Create router, I don't think the SDK needs to follow the same convention and I feel the `RouterVersion.create` would be more standard.

Reply:
Yep that sounds logical. I'll shift the `create_version` to be a `RouterVersion` class method then. Maybe I'll rename it to `create` too so that `RouterVersion.create` would mirror `Router.create` in the SDK.

Reply:
> However, this would also now mean that we'll have to explicitly specify the router_id of the router that we'll like to create this new version for as an explicit argument in the RouterVersion.create method, since RouterConfig instances don't contain any ids.

That sounds logical to me.